pkg/operator/ceph/object/controller.go

```diff
@@ -483,3 +484,10 @@ func (r *ReconcileCephObjectStore) startMonitoring(objectstore *cephv1.CephObjec
 	logger.Info("starting rgw healthcheck")
 	go rgwChecker.checkObjectStore(r.objectStoreChannels[objectstore.Name].stopChan)
 }
+
+func buildStatusInfo(cephObjectStore *cephv1.CephObjectStore) map[string]string {
+	m := make(map[string]string)
+	m["endpoint"] = buildDNSEndpoint(BuildDomainName(cephObjectStore.Name, cephObjectStore.Namespace), cephObjectStore.Spec.Gateway.Port)
```

Comment:
how about a comment that shows an example of the endpoint?

Reply:
What if `SecurePort` is set instead of `Port`?

Reply:
`Port` cannot be empty, it is always mandatory and `SecurePort` is optional.